docs/docs/running-hello-world.md

```diff
@@ -1,55 +1,56 @@
-# Compiling and Running your First Enclave
+# Compiling and running your first enclave
 
 ## Introduction
 
-This tutorial describes how to compile and run the sample hello world app, which you can find in the 
-[Conclave tutorials repository](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world). This app contains the `ReverseEnclave`, 
-which reverses a string provided by the client and passes it back.
+This tutorial describes how to compile and run the sample hello world app, which you can find in the
+[Conclave tutorials repository](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world).
+This app contains the `ReverseEnclave` function, which reverses a string provided by the client and passes it back.
 
-We are first going to run the app in [mock mode](enclave-modes.md),
-which is the simplest (and least realistic) of the enclave modes.
-We will explore running in other modes [later on](#beyond-mock-mode).
+Follow the instructions to run the app in [mock mode](enclave-modes.md). After running it in mock mode, you
+can proceed to [run the app in other modes](#beyond-mock-mode).
 
-If you get stuck, join our [discord server](https://discord.com/invite/dDaBGqGPre) and ask for help!
+If you get stuck at any step, please [talk to us on Discord](https://discord.gg/zpHKkMZ8Sw).
 
 ## Prerequisites
 
-* You need JDK 17 installed in order to build and run the app.
 * This tutorial assumes you've read and understood the [conceptual overview](enclaves.md)
```

Comment:
enclaves.md is actually a big page, and it doesn't make sense to make it a prerequisite of running hello world. It defeats the whole purpose of trying to get a new user comfortable with Conclave. I would actually say this page would be further reading after they've run the sample.

Reply:
I agree. I would even remove the whole thing. As a user I would really want to avoid verbose words and get to the meat of getting the app up and running (and then, only after that, I would want to understand details

Reply:
Removed the prerequisite.